### DISCUSSION TITLE
Can't find packages like datamash or samtools when installing on M1 in arm64 conda

### DISCUSSION THREAD

**Discussion by kaysagit** [biostars/biostar-handbook-issues#343] [Q&A]
Hi,

i downloaded miniconda for Apple Silicon and tried to download the packages with python versions > 3.7 but it still cant install packages like datamash or hisat2.

Did anybody manage to install everything on an Apple Silicon?

**Comment by ialbert** [comment]
check the latest installation instructions in the book,

the installation is all automatic, and you don't need to do it by hand - there are step by step guides but that should not be needed on MacOS

**Reply by kaysagit** [reply]
Yes, was trying older setup code, now everything works, thank you!
